src/clim2parquet/__init__.py

```diff
@@ -0,0 +1,108 @@
+# SPDX-FileCopyrightText: 2025-present Imperial College of Science and Technology
+#
+# SPDX-License-Identifier: MIT
+
+"""
+Convert country climate data to Parquet files.
+
+This package provides functionality to find and convert country climate data,
+assumed to be stored as CSVs, to Parquet files.
+"""
+
+from pathlib import Path
+from typing import Optional
+
+from . import tools
```

Comment:
I believe relative imports are frowned on https://docs.astral.sh/ruff/rules/relative-imports/ - if you add the `TID` rule you'll get this one in ruff

```suggestion
from clim2parquet import tools
```

Reply:
Odd that it wasn't flagged - I have it enabled, copied over from `taskwait`.

Reply:
I think this was an 'explicit relative import' and supposedly okay by ruff.